lib/common/global_header/global_header.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+
+import '../../constants/common_constants.dart';
+import '../../constants/global_header/global_header_constants.dart';
+import '../format_text.dart';
+
+class GlobalHeader extends StatelessWidget implements PreferredSizeWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      child: addAppBarTextWidget(), 
+      height: preferredSize.height,
+      padding: EdgeInsets.only(left: 25.0, right: 15.0),
+      color: Colors.transparent
+    );
+  }
+
+  @override
+  Size get preferredSize => const Size.fromHeight(50.0);
+}
+
+Row addAppBarTextWidget() {
+  return Row(
+    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+    children: <Widget>[
+      FormatText(
+        text: appTitle, 
+        fontSize: globalHeaderFontSize,
+      ), 
+      addAppBarActions()
+    ]
+  );
+}
+
+IconButton addAppBarActions() {
+  return IconButton(
+    icon: Icon(
+      Icons.crop_free, 
+      color: Colors.grey
+    ), 
+    onPressed: () {
+      print("Pairings: Pressed global header action");
```

Comment:
Based on @laiiibah's mockup, it would bring up the report screen.
> ![image](https://user-images.githubusercontent.com/23146829/87327742-b28d7980-c4f1-11ea-9744-8aa059f62007.png)

Reply:
So the print statement would need to be eventually changed to open a small menu instead?

Reply:
Looks good
!

Reply:
@AyshaAraien, eventually it would need to be changed to open up a new screen. We can talk about which screen it opens at standup!